app/js/features/metadata/reducer.js

```diff
@@ -85,6 +86,16 @@ function reducer(state = DEFAULT_STATE, action) {
         ...state, offers: newOffers
       };
     }
+    case USER_RATING_SUCCEEDED:
```

Comment:
feels weird to reply to an event from escrow in this specific reducer,
I had to do the same with escrow and offer
=> I leave the reducer as they were but move the mix in the reducer.

I feels this is because we have the wrong structure. I am not sure what it should be to avoid those mixed.
Any idea?

Reply:
Not sure. We either have escrow function calls in the metadata saga or have escrow constants in the metadata reducer. There is no good way around it I think. 
I'd leave it like that for now as we don,t have a solution and I don't want to move stuff around just for the sake of it.

Reply:
Sure I was just asking to know if you have a better idea but you are like me :joy: